mobile-browser-based-version/src/logic/communication/client.ts

```diff
@@ -1,57 +1,53 @@
 import * as tf from '@tensorflow/tfjs'
+import { Task } from '../task_definition/base/task'
+import { TrainingInformant } from '../training/training_informant'
 
-export class Client {
-  serverURL: any;
-  task: any;
-  constructor (serverURL, task) {
+export abstract class Client {
+  serverURL: string;
+  task: Task;
+  constructor (serverURL: string, task: Task) {
     this.serverURL = serverURL
     this.task = task
   }
 
+  /**
+   * Init holder for federated, we need to fetch latest server round
+   * and model before training.
+   */
+  async updateModelAndRoundFromServer () {}
```

Comment:
I'm not sure why this is in `Client`, as this is typically not used by `DecentralizedClient`

Reply:
you are right, it's also no longer used due to refactoring so I will delete it.